pkg/operator/mocks/muo/muo.go

```diff
@@ -1,96 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
What is the reason for this file removal? I believe it has a purpose as it is present in the master.

Reply:
Hitting a small snag trying to consolidate some of the requested changes. Stay tuned :sweat_smile: